product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: Step 4 - Installing the PGD software
+navTitle: Installing PGD software
+deepToC: true
+---
+
+## Installing the PGD software
+
+With the repositories configured, you can now install the Postgres Distributed software. 
+These steps must be carried out on each host.
+
+* Install the PGD packages which include a server specific BDR package and generic PGD proxy and cli packages. (`edb-bdr5-postgresversion`, `edb-pgd5-proxy`, and `edb-pgd5-cli`)
+* Ensure the Postgres database server has been initialized and started.
+* Add the BDR extension to the shared_preload_libraries setting in `postgresql.conf`.
+* Set the `wal_level` GUC variable to `logical` in `postgresql.conf`.
+* Turn on commit timestamp tracking by setting `track_commit_timestamp` to `'on'` in `postgresql.conf`.
+* Raise the maximum worker processes to 16 by setting `max_worker_processes` to `'16'` in `postgresql.conf`.
+* Set a password on the EnterprisedDB/Postgres user.
+* Add rules to `pg_hba.conf` to allow nodes to connect to each other.
+* Add a `.pgpass` file to allow nodes to authenticate each other.
+* Restart the server.
```

Comment:
suggest adding something like "... and check the postgres log to and verify the server has started correctly with the modified settings"